.gitignore

```diff
@@ -1,11 +1,6 @@
 ## Original ignores
 autogpt/keys.py
-autogpt/*json
-autogpt/node_modules/
-autogpt/__pycache__/keys.cpython-310.pyc
-autogpt/auto_gpt_workspace
-package-lock.json
-*.pyc
+autogpt/*.json
 auto_gpt_workspace/*
```

Comment:
@ntindle Is `autogpt/auto_gpt_workspace` not a duplicate of `auto_gpt_workspace/*` ?